docs/site/themes/template/layouts/index.html

```diff
@@ -37,6 +38,18 @@ <h3>Shape the Future</h3>
 			<div class="wrapper">
 				<h2>A complete and customizable Kubernetes platform</h2>
 				<p>Tanzu Community Edition includes the same open source software used in Tanzu commercial offerings.</p>
+				<div class="buttons">
+					<button class="button navy-blue modal-video"
+						data-target="#videoModal"
```

Comment:
@michow I was hoping to make this button a partial that accepted params (title, id), but stopped here to be mindful of time.